openapi/endpoints/funding-source/models/create-funding-source-eoa.yaml

```diff
@@ -0,0 +1,20 @@
+type: object
+required:
+  - accountId
+  - fundingAddress
+  - fundingSourceType
+properties:
+  accountId:
+    type: string
+    description: The id of the cardholder account to add this funding source to.
+    example: 65a7e8ef0d230d0a6bf755e07d39eb02
+  fundingAddress:
+    type: string
+    description: The address to claim as a funding source. This can be a smart contract implementing ERC-1271 or an EOA address.
+    example: 0x1234567890123456789012345678901234567890
+  fundingSourceType:
+    type: string
+    description: The type of funding source to create.
+    enum:
+      - ImmersvePolygonMumbaiUSDC
```

Comment:
is this valid? I thought we had them as separate enum names, but the same value

Reply:
In amethyst they are different values too